### PR TITLE
Shared clipboard

### PR DESCRIPTION
**Pull request by rgantzos**
This feature allows you to copy items from one paint editor and paste them into another project's paint editor.

### CODE REVIEW

File: features/shared-clipboard/data.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "title": "Shared Clipboard",
+  "description": "Allows you to copy and paste items between paint editors in different projects.",
+  "credits": [
+    { "username": "rgantzos", "url": "https://scratch.mit.edu/users/rgantzos/" }
+  ],
+  "type": ["Editor"],
+  "tags": ["New"],
+  "scripts": [{ "file": "script.js", "runOn": "/projects/*" }],
+  "similar": []
```
Comment (StioStudio) [inline features/shared-clipboard/data.json:10]: Is similar needed?

File: features/shared-clipboard/script.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+var interval;
```
Comment (StioStudio) [inline features/shared-clipboard/script.js:1]: Why use var? Var is getting more and more left behind. I recommend let

Comment (StioStudio) [reply features/shared-clipboard/script.js:1]: Btw you don't use interval anywhere else

File: features/shared-clipboard/script.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        });
+        lastClipboard = currentClipboard;
+      } else if (
+        JSON.stringify(lastClipboard) !== JSON.stringify(loadedClipboard)
+      ) {
+        ScratchTools.Scratch.scratchPaint().clipboard.items = loadedClipboard;
+        lastClipboard = loadedClipboard;
+      }
+    }
+    setTimeout(fetchClipboardData, 500);
```
[the remark this answers is not in the document]
Comment (rgantzos) [reply features/shared-clipboard/script.js:33]: To let a little bit of time in between each call.

**Review by StioStudio — APPROVED** [review]
Good feature

**Review by StioStudio — APPROVED** [review]

**Pull request merged**